CleverTapSDK/EventDatabase/CTEventDatabase.h

```diff
@@ -10,27 +10,32 @@
 #import <sqlite3.h>
 #import "CleverTapEventDetail.h"
 #import "CTClock.h"
+#import "CTDispatchQueueManager.h"
 
 @interface CTEventDatabase : NSObject
 
-+ (instancetype)sharedInstance;
-- (instancetype)initWithClock:(id <CTClock>)clock;
++ (instancetype)sharedInstanceWithDispatchQueueManager:(CTDispatchQueueManager*)dispatchQueueManager;
+- (instancetype)initWithDispatchQueueManager:(CTDispatchQueueManager*)dispatchQueueManager
```

Comment:
i think this method can be private since it is only called from the sharedInstance method(and tests)